packages/web/src/store/selectors/websiteResults.ts

```diff
@@ -94,6 +108,12 @@ export const selectors = {
           vulnerable: identifiedPackages.filter((pkg) => !!pkg.vulnerable).length,
           outdated: identifiedPackages.filter((pkg) => !!pkg.outdated).length,
         },
+        scriptsCount: scanResult?.scan?.scanResult?.processedScripts?.filter(
+          (script) => script.status === 'processed'
```

Comment:
IMO, we should either remove the `processed` check here or add it below to size calculation for consistency.

Reply:
from my perspective we should only consider wabpack-related scripts for stats